corehq/ex-submodules/pillowtop/reindexer/reindexer.py

```diff
@@ -144,19 +145,21 @@ def reindex(self):
         pillow_logging.info("Processed %s docs", total_docs)
 
 
-def clean_index(es, index_info):
-    if es.indices.exists(index_info.index):
-        es.indices.delete(index=index_info.index)
+def clean_index(index_name):
+    if manager.index_exists(index_name):
+        manager.index_delete(index=index_name)
 
 
-def prepare_index_for_reindex(es, index_info):
+def prepare_index_for_reindex(adapter):
+    es = get_client()
+    index_info = index_info_from_adapter(adapter)
     initialize_index_and_mapping(es, index_info)
```

Comment:
the `es` param is unused so can be set to None and remove the `es` var above
```suggestion
    initialize_index_and_mapping(None, index_info)
```

Reply:
This I will do when I work on removing es param totally.